mypyc/test-data/irbuild-str.test

```diff
@@ -137,6 +137,38 @@ L4:
 L5:
     unreachable
 
+[case testStrStartswithEndswithTuple]
+from typing import Tuple
+
+def do_startswith(s1: str, s2: Tuple[str, ...]) -> bool:
+    return s1.startswith(s2)
```

Comment:
I'm more interested in what happens when `startswith` is given a tuple literal argument (e.g. `return s1.startswith(('x', 'y'))`), since this is probably the most common use case. Can you test also this? It would be good to have also a run test for this.